src/plugins/run/unwrap-images.ts

```diff
@@ -51,5 +55,7 @@ function applicable(node: Parent, inLink?: boolean): boolean | null {
     }
   }
 
+  if (nonwhitespaceChildren > 1) return false;
```

Comment:
seems like we could move it to the loop's body to abort the iteration early?

Reply:
yes good call